package.json

```diff
@@ -5,8 +5,13 @@
   "main": "gulpfile.js",
   "scripts": {
     "build": "gulp",
-    "start": "node server.js"
+    "dev": "gulp dev",
+    "start": "node -e 'require(\"./server.js\")()'"
   },
+  "files": [
+    "dist",
+    "server.js"
+  ],
```

Comment:
This block tells `npm` that if we publish this as an NPM package, these are the only two things that need to be included. Only the built content and a server are necessary to run the site.